lua/colorbox/controller.lua

```diff
@@ -138,6 +140,26 @@ M._parse_args = function(args)
   return opts
 end
 
+M.shuffle = function()
+  local ColorNamesList = runtime.colornames()
+  if #ColorNamesList > 0 then
+    local logger = logging.get("colorbox")
+    local random_index = math.random(1, #ColorNamesList)
```

Comment:
hi @marcelarie , please use `num.random` api, it has better pseudo-random implementation.